etc/openqa/workers.ini

```diff
@@ -23,3 +23,8 @@
 
 #[openqa.example.host]
 # SHARE_DIRECTORY = /var/lib/openqa/example
+# In case caching of assets and tests needs to be enabled
+# CACHEDIRECTORY and TESTPOOLSERVER need to be enabled
+# CACHEDIRECTORY = /var/lib/openqa/cache
+# TESTPOOLSERVER = rsync://deimos.suse.de/openqa-tests
```

Comment:
My approach would sound like:

I think this is hard to understand without context. Can you add more descriptions what TESTPOOLSERVER stands for?